tests/integration/views/surveys/help/test_surveys_help_info_about_the_ons.py

```diff
@@ -37,13 +33,9 @@ def set_flask_session(self):
             mock_session["help_ru_ref"] = "49900000001F"
 
     @requests_mock.mock()
-    @patch("frontstage.controllers.party_controller.get_business_by_ru_ref")
```

Comment:
The mocking in here is pretty terrible, I have removed ones I can see it clearly doesn't need, but I haven't got them all